tkg/test/tkgctl/shared/e2e_common_cc.go

```diff
@@ -148,6 +148,11 @@ func E2ECommonCCSpec(ctx context.Context, inputGetter func() E2ECommonCCSpecInpu
 	})
 
 	AfterEach(func() {
+		framework.PrintPodsLogs(mngProxy, "capa-system")
```

Comment:
Is there a way to check only when the test case fails, we print the pod logs?

Reply:
Ok, it will be corrected, the output log will be a lot each time